content/cloud/faq.md

```diff
@@ -42,8 +42,8 @@ The following are the quota for users based, stratified by the three [Subscripti
 | Key value maximum keys* | 1,024 | 1,024 | Contact us
 | Key value store count | 5 | 100 | Contact us
 | Key value storage size (GB) | 1 | 2 | Contact us
-| SQLite database count | 1 | 1 |  Contact us
-| SQLite database size (GB) | 1 | 2 | Contact us
+| SQLite Database count | 1 | 1 |  Contact us
```

Comment:
I don't think this is the brand name, just "database" in its "the concept that Fermyon Cloud SQLite Database implements" which is why it wasn't NoOps before

```suggestion
| SQLite database count | 1 | 1 |  Contact us
```